src/harness/testcases/spectrum_inquiry_testcase.py

```diff
@@ -391,6 +391,90 @@ def test_WINNF_FT_S_SIQ_16(self):
       self.assertEqual(resp['response']['responseCode'], 0)
 
   @winnforum_testcase
+  def test_WINNF_FT_S_SIQ_17(self):
+    """This test ensures that when a message with two requests, 
+    both for PAL channels, is sent with valid parameters, 
+    the SAS sends a successful response for both the requests.
+    Response Code must be 0 for all CBSDs
+    """
+
+    # Load the Data
+    pal_low_frequency = 3550000000.0
+    pal_high_frequency = 3560000000.0
+    registration_request = []
+    ppa_ids = []
+    for device_filename, ppa_filename in zip(('device_a.json', 'device_c.json'),
+                                             ('ppa_record_0.json', 'ppa_record_1.json')):
+      device = json.load(
+        open(os.path.join('testcases', 'testdata', device_filename)))
+      pal_record = json.load(
+        open(os.path.join('testcases', 'testdata', 'pal_record_0.json')))
+      ppa_record = json.load(
+        open(os.path.join('testcases', 'testdata', ppa_filename)))
+      ppa_record, pal_record = makePpaAndPalRecordsConsistent(ppa_record,
+                                                                  [pal_record],
+                                                                  pal_low_frequency,
+                                                                  pal_high_frequency,
+                                                                  device['userId'])
+
+      self._sas_admin.InjectZoneData({"record": ppa_record})
```

Comment:
Thanks for all your comments! I recently created PR to change the zoneData to record for consistency https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/blob/master/src/harness/sas_interface.py#L254 and to match with the Test API Doc https://docs.google.com/document/d/19YbeQsSzm5kCIOeVzsF8piqI0Lizy3p345aiVGf-Hlg/edit#heading=h.fbgibud28jgj